packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/GroupForm.tsx

```diff
@@ -165,6 +169,12 @@ const GroupForm = ({
   };
 
   const addRequirementSubForm = () => {
+    if (requirementSubForms.length <= MAX_REQUIREMENTS) {
+      setOverRequirementLimit(false);
+    }
+    if (requirementSubForms.length >= MAX_REQUIREMENTS) {
+      setOverRequirementLimit(true);
+    }
```

Comment:
this will not work in update mode

Reply:
Added [this](https://github.com/hicommonwealth/commonwealth/pull/5503/commits/45ded34715533121c0f6dd5e021103b932c2814e) commit to fix that